ui/src/tempVars/components/TemplateVariableEditor.tsx

```diff
@@ -384,6 +431,21 @@ class TemplateVariableEditor extends PureComponent<Props, State> {
   }
 }
 
-const mapDispatchToProps = {notify: notifyActionCreator}
+const mapDispatchToProps = {
+  notify: notifyActionCreator,
+}
+
+const mstp = (state, props) => {
```

Comment:
so we seem to be split between `mdtp` / `mapDispatchToProps` and `mstp` / `mapStateToProps` (search for them). In the cases I saw, there was internal consistence between `mdtp` -> `mstp` and `mapDispatchToProps` -> `mapStateToProps`  which means i think this should be changed to `mapStateToProps`

Reply:
Sounds good